source/Table/Table.js

```diff
@@ -453,13 +453,15 @@ export default class Table extends PureComponent {
       a11yProps['aria-describedby'] = id;
     }
 
+    a11yProps.key = "Row" + rowIndex + "-" + "Col" + columnIndex;
+    a11yProps.className = cn('ReactVirtualized__Table__rowColumn', className);
+    a11yProps.style = style;
+    a11yProps.title = title;
```

Comment:
Can you move all these props to object literal and rename `a11yProps` to `cellProps`?